.github/workflows/run-forge.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
           python-version: '3.9'
       - name: Install python deps
         if: ${{ env.FORGE_ENABLED == 'true' && env.WRAPPER_KILLSWITCH == 'false' }}
-        run: pip3 install -r testsuite/requirements.txt
+        run: pip3 install click==8.1.3 psutil==5.9.1
```

Comment:
why is this better?

Reply:
Its transitional, with this change all workflows (no matter the merge base) will look for a testsuite/requirements.txt file

If that file doesnt exist (which it wont until you have rebased) it will fail all PR until then :yikes:

So what this does is just install the deps for now, once sufficient time has gone by, can switch this back to requirements txt, because that will move in tandem with the forge py source

We can make this process faster and force changes like this in the future if we adopt a minimum revision check like I'm trying in https://github.com/aptos-labs/aptos-core/pull/3108